packages/flutter/lib/src/widgets/overscroll_indicator.dart

```diff
@@ -98,19 +98,21 @@ class GlowingOverscrollIndicator extends StatefulWidget {
   _GlowingOverscrollIndicatorState createState() => new _GlowingOverscrollIndicatorState();
 
   @override
-  void debugFillDescription(List<String> description) {
-    super.debugFillDescription(description);
-    description.add('$axisDirection');
+  void debugFillProperties(List<DiagnosticsNode> description) {
+    super.debugFillProperties(description);
+    description.add(new EnumProperty<AxisDirection>('axisDirection', axisDirection));
+    String showMessage;
     if (showLeading && showTrailing) {
-      description.add('show: both sides');
+      showMessage = 'both sides';
     } else if (showLeading) {
-      description.add('show: leading side only');
+      showMessage = 'leading side only';
     } else if (showTrailing) {
-      description.add('show: trailing side only');
+      showMessage = 'trailing side only';
     } else {
-      description.add('show: neither side (!)');
+      showMessage = 'neither side (!)';
     }
-    description.add('$color');
+    description.add(new StringProperty('show', showMessage));
```

Comment:
As most of the StringProperty cases with `quoted: false` were these `message` properties where a message was formatted to look consistent with other regular properties even though the object lacked a concrete property matching the concept I've created a trivial MessageProperty class that captures the concept of a message that should be displayed in a property like manner with a separate name and description.